ntbs-service/Services/AuthorizationService.cs

```diff
@@ -79,17 +80,21 @@ async Task SetPadlockForBannerAsync(ClaimsPrincipal u, NotificationBannerModel b
 
             if (_userPermissionsFilter.Type == UserType.NationalTeam)
             {
-                return (PermissionLevel.Edit,
+                return (await _userService.GetUser(user)).IsReadOnly

```

Comment:
Here, we're getting the user several times. I think it would be better to pull this out so that we only get it once.